File: TurnosBackend/Data/Models/ClientXTurnXUserXService.cs
using System;
using System.Collections.Generic;

namespace Data.Models;

public partial class ClientXTurnXUserXService
{
    public int IdClient { get; set; }

    public int IdTurn { get; set; }

    public int IdUser { get; set; }

    public int IdService { get; set; }

    public virtual Client? Client { get; set; } = null!;

    public virtual Turn? Turn { get; set; } = null!;

    public virtual User? User { get; set; } = null!;

    public virtual Service? Service { get; set; } = null!;
}

File: TurnosFrontend/src/app/models/clientXTurnXUserXService.ts
import { Client } from "./client";
import { Turn } from "./turn";
import { User } from "./user";
import { Service } from "./service";

export interface ClientXTurnXUserXService {
  idClient: number;
  idTurn: number;
  idUser: number;
  idService: number;
  client: Client;
  turn: Turn;
  user: User;
  service: Service;
}